### PR TITLE
An attempt at implementing the filter_gamesplayed feature!

### PR DESCRIPTION
**Pull request by wanderrful**
Partner request for https://github.com/odota/ui/pull/1324

### CODE REVIEW

File: routes/spec.js [hunk 1]
```diff
@@ -1447,6 +1448,7 @@ Please keep request rate to approximately 3/s.
             });
             const result = Object.keys(heroes)
               .map(k => heroes[k])
+              .filter(hero => !req.queryObj.having || hero.games_played >= Number(req.query.having))
```
Comment (howardchung) [inline routes/spec.js]: use queryObj in the second one too

File: routes/params.js [hunk 2]
```diff
@@ -145,6 +145,13 @@ module.exports = {
     required: false,
     type: 'string',
   },
+  minimumPlayedParam: {
+    name: 'games_played',
```
Comment (howardchung) [inline routes/params.js]: if you named it 'having' below then also name it 'having' here

File: routes/params.js [hunk 3]
```diff
@@ -145,6 +145,13 @@ module.exports = {
     required: false,
     type: 'string',
   },
+  minimumPlayedParam: {
```
Comment (howardchung) [inline routes/params.js]: and call it havingParam

**Comment by wanderrful** [comment]
Right sorry -- I'm not familiar with this routing library that you're using here so I'm just sort of making it up as I go.  I've incorporated the changes you asked.

File: routes/spec.js [hunk 4]
```diff
@@ -1447,6 +1448,7 @@ Please keep request rate to approximately 3/s.
             });
             const result = Object.keys(heroes)
               .map(k => heroes[k])
+              .filter(hero => !req.queryObj.having || hero.games_played >= Number(req.queryObj.having))
```
Comment (howardchung) [inline routes/spec.js]: looks like the field is "games" as the sort indicates below.  I had it wrong originally

Comment (howardchung) [reply routes/spec.js]: instead of games_played

**Comment by wanderrful** [comment]
I've edited the "games_played" and made it into "games" for you, as you asked.  Hope this does the trick!

**Pull request merged**